js.test

```diff
@@ -0,0 +1,14 @@
+var movies = ["The Shawshank Redemption",
+"The Godfather",
+"The Dark Knight",
+"12 Angry Men",
+"The Lord Of The Rings",
+"Pulp Fiction",
+"The Good, the Bad and the Ugly",
+"Forrest Gump"];
+
+console.log(movies);
+///NUmber of words
+const NumWords = ["3 words", "2 words", "3 words", "3 words", "5 words", "2 words", "7 words", "2 words"];
```

Comment:
Needless to say, we wanted you to calculate the number of words with code. That might seem like a difficult, challenge, right? Let us see, using our friend Google: type in `javascript number of words in string` and get [this](https://stackoverflow.com/questions/18679576/counting-words-in-string) as a result, which contains the following simple solution: `str.split(' ').length;`